models.py

```diff
@@ -252,11 +252,13 @@ def update(self, **kwargs):
             if attribute in self.unique_attributes:
                 raise ValueError(f'{attribute} can\'t be be updated.')
             if hasattr(self, attribute):
-                setattr(self, attribute, value)
                 if 'enable' in attribute:
```

Comment:
Now I understood that the code implicitly gets both 'enable' and 'enabled', but also any other string that may contain 'enable' in it. Maybe a more explicit match would be an exact one like this:

```suggestion
                if attribute in ('enable', 'enabled'):
```